nautobot_chatops_ipfabric/worker.py

```diff
@@ -513,10 +524,24 @@ def pathlookup(
         ]
     )
 
-    # only supported in IP Fabric OS version 4.0+
+    # diagrams for 4.0 - 4.2 are not supported due to attribute changes in 4.3+
     try:
-        if ipfabric_api.validate_version(ge, 4.0):
-            raw_png = ipfabric_api.get_pathlookup(src_ip, dst_ip, src_port, dst_port, protocol, snapshot_id)
+        os_version = ipfabric_client.os_version
+        if os_version and ge(os_version, "4.3"):
+            ipfabric_diagram_client = IPFDiagram(
+                base_url=settings.PLUGINS_CONFIG["nautobot_chatops_ipfabric"].get("IPFABRIC_HOST"),
+                token=settings.PLUGINS_CONFIG["nautobot_chatops_ipfabric"].get("IPFABRIC_API_TOKEN"),
+                verify=settings.PLUGINS_CONFIG["nautobot_chatops_ipfabric"].get("IPFABRIC_VERIFY"),
```

Comment:
it's weird that having the client initialized above we have to do it again, can't we reuse the session?